public_dropin_environments/python3_keras/fit.sh

```diff
@@ -9,14 +9,11 @@ export weights="${INPUT_DIRECTORY}/weights.csv"
 CMD="drum fit --target-type ${TARGET_TYPE} --input ${X} --num-rows ALL --output ${ARTIFACT_DIRECTORY} \
 --code-dir ${CODEPATH} --verbose"
 
-if [ -n "${UNSUPERVISED}" ]; then
-  CMD="${CMD} --unsupervised "
-else
+if [ "${TARGET_TYPE}" != "anomaly" ]; then
```

Comment:
@4dahalibut  we don't want to have to pass y to transform fit, right? @yakov-g is there a non-clunky way to look for the `y` env var and only add the `--target-csv` command for transform if it's present? otherwise everything looks good to me

Reply:
I can suggest the following.
Now drum can take args directly from env vars: TARGET_TYPE, CLASS_LABELS, some others.
 (Check this PR - I'm removing checl for POSITIVE/NEGATIVE_CLASS_LABELS from fit.sh)

What we can do is: 
- add support for TARGET_CSV/TARGET env vats in drum;
- add setting TARGET_CSV env var in the APP, for the target types where it is needed
- and then we can completely remove: 
```
if [ "${TARGET_TYPE}" != "anomaly" ]; then
  export y="${INPUT_DIRECTORY}/y.csv"
  CMD="${CMD} --target-csv ${y}"
fi
```

I can open a ticket for now, to do it later

Reply:
https://datarobot.atlassian.net/browse/RAPTOR-4387

Reply:
So guys, let's merge it for now: @jlpolit @4dahalibut

Reply:
ok, I think that's alright- definitely tag me on the related Datarobot PR